Perf.DiffPlex/Perf.DiffPlex.csproj

```diff
@@ -10,8 +10,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" Version="0.13.12" />
-    <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.12" />
+    <PackageReference Include="BenchmarkDotNet" />
```

Comment:
@kingcean  Can you run benchmarks before before and after the changes and add the results to the PR description? I want to see if we have a measurable impact with this change. We may need to make this benchmark more interesting though

Reply:
It would be also good to use this to analyze what else we can improve.